pkg/config/config_test.go

```diff
@@ -360,4 +361,44 @@ var _ = Describe("Config", func() {
 		})
 	})
 
+	Describe("Reload", func() {
+		It("test new config from reload", func() {
+			// Default configuration
+			defaultTestFile := "testdata/containers_default.conf"
+			oldEnv, set := os.LookupEnv("CONTAINERS_CONF")
+			os.Setenv("CONTAINERS_CONF", defaultTestFile)
+			cfg, err := Default()
+			gomega.Expect(err).To(gomega.BeNil())
+			if set {
+				os.Setenv("CONTAINERS_CONF", oldEnv)
+			} else {
+				os.Unsetenv("CONTAINERS_CONF")
+			}
+
+			// Reload from new configuration file
+			testFile := "testdata/temp.conf"
+			content := `[containers]
+env=["foo=bar"]`
+			err = ioutil.WriteFile(testFile, []byte(content), os.ModePerm)
+			defer os.Remove(testFile)
+			gomega.Expect(err).To(gomega.BeNil())
+			oldEnv, set = os.LookupEnv("CONTAINERS_CONF")
+			os.Setenv("CONTAINERS_CONF", testFile)
+			newCfg, err := Reload()
```

Comment:
Can you change this to _, err := Reload
Then after it is successful, do

newcfg := Default()

Then we know that you modified the Default() function and not just the Reload function.

Reply:
fixed